Problem1/spec.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "operation": "modify-overwrite-beta",
+    "spec": {
+      "orders": {
+        "*": {
+          "customer": {
+            "fullName": "=concat(@(1,firstName),' ',@(1,lastName))",
+            "address": "=concat(@(0,street),', ',@(0,city),', ',@(0,state),' ',@(0,zip))"
+          },
+          "items": {
+            "*": {
+              "inverse": "=divide(1,@(1,quantity))",
+              "newprice": "=divide(@(1,price),@(1,inverse))",
+              "price": "=toDouble(@(1,price))"
+            }
+          }
+        }
+      }
+    }
+  },
+  {
+    "operation": "shift",
+    "spec": {
+      "orders": {
+        "*": {
+          "*": "orders[&1].&",
+          "items": {
+            "*": {
+              "*": "orders[&3].items[&1].&",
+              "newprice": "orders[&3].&"
```

Comment:
It should be total instead of newprice

Reply:
Corrected the output and changed the name to total instead of newprice.